### ISSUE TITLE
Github Action for CI and CD

### ISSUE THREAD

**Issue by jermnelson** [sul-dlss-labs/spoc#5]
Create Github Actions for continuous integration test and deployment to AWS.

**Comment by jgreben** [comment]
@jermnelson for the CD portion of this, do we know the AWS secrets?
```
env:
  AWS_REGION: MY_AWS_REGION                   # set this to your preferred AWS region, e.g. us-west-1
  ECR_REPOSITORY: MY_ECR_REPOSITORY           # set this to your Amazon ECR repository name
  ECS_SERVICE: MY_ECS_SERVICE                 # set this to your Amazon ECS service name
  ECS_CLUSTER: MY_ECS_CLUSTER                 # set this to your Amazon ECS cluster name
  ECS_TASK_DEFINITION: MY_ECS_TASK_DEFINITION # set this to the path to your Amazon ECS task definition
                                               # file, e.g. .aws/task-definition.json
  CONTAINER_NAME: MY_CONTAINER_NAME           # set this to the name of the container in the
                                               # containerDefinitions section of your task definition

```
https://docs.github.com/en/actions/guides/deploying-to-amazon-elastic-container-service#creating-the-workflow

**Comment by jgreben** [comment]
Now Blocked by #52

**Comment by jgreben** [comment]
Fixed by #51
Fixed by #60
